src/console/console_cmd.c

```diff
@@ -172,9 +212,24 @@ int console_cmd_lose(game_state *gs, int argc, char **argv) {
         object *har_obj = game_state_find_object(gs, game_player_get_har_obj_id(player));
         if(!har_obj)
             return 1;
-        har *har = object_get_userdata(har_obj);
-        har->health = 1;
-        return 0;
+
+        rec_assertion op;
```

Comment:
please zero this struct before filling its fields.
i believe this is currently leaking stack memory content in uninitialized padding bytes.

Reply:
oh never mind the serializer actually goes and writes only the initalized bits.
i'm too used to a project where serialization is all done by `memcpy`